cucumber-scala/src/main/resources/META-INF/plugin.xml

```diff
@@ -20,9 +20,9 @@
       <ul>
       <li>2024.1: Release for IntelliJ 2024.1 (build 241.14494.240)
```

Comment:
Instead of changing this entry in the changelog there should be a new list item above it for
```
      <li>2024.2: Release for IntelliJ 2024.2 (build 242.20224.300)
          <ol>
            <li>Scala plugin 2024.2.20</li>
            <li>Gherkin plugin 242.20224.159</li>
            <li>cucumber-scala 8.23.0</li>
          </ol>
      </li>
```